app/src/main/java/com/duckduckgo/app/onboarding/ui/OnboardingActivity.kt

```diff
@@ -0,0 +1,168 @@
+/*
+ * Copyright (c) 2018 DuckDuckGo
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.duckduckgo.app.onboarding.ui
+
+import android.arch.lifecycle.Observer
+import android.arch.lifecycle.ViewModelProviders
+import android.content.Context
+import android.os.Bundle
+import android.support.annotation.ColorInt
+import android.support.annotation.ColorRes
+import android.support.v4.app.Fragment
+import android.support.v4.app.FragmentManager
+import android.support.v4.app.FragmentPagerAdapter
+import android.support.v4.view.ViewPager.OnPageChangeListener
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import com.duckduckgo.app.browser.R
+import com.duckduckgo.app.global.DuckDuckGoActivity
+import com.duckduckgo.app.global.ViewModelFactory
+import com.duckduckgo.app.global.view.ColorCombiner
+import com.duckduckgo.app.home.HomeActivity
+import kotlinx.android.synthetic.main.activity_onboarding.*
+import javax.inject.Inject
+
+
+class OnboardingActivity : DuckDuckGoActivity() {
+
+    @Inject lateinit var viewModelFactory: ViewModelFactory
+    private lateinit var viewPageAdapter: PagerAdapter
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_onboarding)
+        configurePager()
+
+        viewModel.viewState.observe(this, Observer<OnboardingViewModel.ViewState> {
+            it?.let { render(it) }
+        })
+    }
+
+    private val viewModel: OnboardingViewModel by lazy {
+        ViewModelProviders.of(this, viewModelFactory).get(OnboardingViewModel::class.java)
+    }
+
+    override fun onResume() {
+        updateColor(viewPageAdapter.color(this, viewPager.currentItem))
+        super.onResume()
+    }
+
+    private fun render(viewState: OnboardingViewModel.ViewState) {
+        if (viewState.showHome) {
+            showHome()
+        }
+    }
+
+    fun onContinueClicked(view: View) {
+        val next = viewPager.currentItem + 1
+        if (next < viewPager.adapter!!.count) {
+            viewPager.currentItem = next
```

Comment:
If you use the setter here you can pass true to make it scroll.

`viewPager.setCurrentItem(next, true)`